### DISCUSSION TITLE
All the issues in one place

### DISCUSSION THREAD

**Discussion by andrey-helldar** [Laravel-Lang/common#118] [Announcements]
@Laravel-Lang/laravel-lang, hi there 👋

I thought it would be convenient to keep all messages from users in one place, so in all repositories I removed the possibility to create local issues and added a redirect to the [laravel-lang/common](https://github.com/Laravel-Lang/common) project.

This will work as follows: the user goes to the "Issue" section of the project he is interested in, clicks on the "New issue" button and on the next screen selects one of the four available issues.

The difference is that at this point the new functionality starts: previously, when creating issue types "bug" or "feature", a local issue was created, but now it will be redirected via URL to the Common project with the necessary fields filled in beforehand.

For each such transition, the "Lang Package Name" field will be automatically filled in and labeled with "bug" or "feature", as well as a [label](https://github.com/Laravel-Lang/common/labels) with the package name to make it easier to separate them.

![image](https://github.com/Laravel-Lang/common/assets/10347617/972e6bed-5d12-4fd6-8fba-814c729dc506)

![image](https://github.com/Laravel-Lang/common/assets/10347617/d6aaf238-dbef-4255-89cc-d6911f5701c9)

![image](https://github.com/Laravel-Lang/common/assets/10347617/50c53c6f-166c-49c7-8a98-66bfd4f6bcd6)

**Comment by mohamedsabil83** [comment]
Good idea @andrey-helldar. That's useful and more organized.
